frontend/src/features/contacts/contact/detail/Person.tsx

```diff
@@ -0,0 +1,197 @@
+import { FormSection } from 'components/common/form/styles';
+import { AddressTypes } from 'constants/addressTypes';
+import { ContactMethodTypes } from 'constants/contactMethodType';
+import { Dictionary } from 'interfaces/Dictionary';
+import {
+  IContactAddress,
+  IContactMethod,
+  IContactOrganization,
+  IContactPerson,
+} from 'interfaces/IContact';
+import * as React from 'react';
+import { Col } from 'react-bootstrap';
+import { FaCircle, FaRegBuilding, FaRegUser } from 'react-icons/fa';
+import { Link } from 'react-router-dom';
+import { phoneFormatter } from 'utils/formUtils';
+
+import * as Styled from '../../styles';
+
+interface IPersonViewProps {
+  person: IContactPerson;
+}
+
+interface ContactInfoField {
+  info: string;
+  label: string;
+}
+
+interface AddressField {
+  address: IContactAddress;
+  label: string;
+}
+
+const PersonView: React.FunctionComponent<IPersonViewProps> = ({ person }) => {
+  const phoneTypes: Dictionary<string> = {};
+  phoneTypes[ContactMethodTypes.PerseMobil] = 'Mobile';
+  phoneTypes[ContactMethodTypes.PersPhone] = 'Home';
+  phoneTypes[ContactMethodTypes.WorkMobil] = 'Work';
+  phoneTypes[ContactMethodTypes.WorkPhone] = 'Fax';
+
+  const personPhoneNumbers: ContactInfoField[] = getContactInfo(person, phoneTypes);
+
+  const emailTypes: Dictionary<string> = {};
+  emailTypes[ContactMethodTypes.WorkEmail] = 'Work';
+  emailTypes[ContactMethodTypes.PerseEmail] = 'Personal';
+  const personEmails: ContactInfoField[] = getContactInfo(person, emailTypes);
+
+  const personAddresses: AddressField[] = person.addresses.reduce(
```

Comment:
Just my opinion and `array.reduce()` is fine here but since you are not using the previous value to compute the next one - wouldn't it be simpler just to use `array.map()` ?